packages/crayons-core/src/utils/types.ts

```diff
@@ -2,6 +2,8 @@ import { VNode } from '@stencil/core';
 
 export type TagVariant = 'standard' | 'avatar';
 
+export type TagState =  'normal' | 'focused' | 'error';
```

Comment:
can we have just have 2 tag States like `normal` and `error` and keep isFocused as a prop to focus on the tag